Translation corrections (#153)
07th-mod/higurashi-console-arcs: Update/some_014.txt

//rそれを傲然と見下ろしながら夏美はゆっくりと歩み寄ると、覗き込むような格好で馬乗りになっていった。そして笑いを噛み締めながら、暁の喉元に手が伸ばされて…！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それを傲然と見下ろしながら夏美はゆっくりと歩み寄ると、覗き込むような格好で馬乗りになっていった。そして笑いを噛み締めながら、暁の喉元に手が伸ばされて…！",

> 			NULL, "Natsumi slowly walked up while looking down at him arrogantly, then staddled him. She reflected on her laughter as she reached for Akira's throat...!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Natsumi slowly walked up while looking down at him arrogantly, then straddled him. She reflected on her laughter as she reached for Akira's throat...!", Line_Normal);
```

//r…喉元に音を立てて、爪が刺さる。そして、血走った目で暁の胸のうちを射抜くように、夏美は深く深く顔を近づけてきた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…喉元に音を立てて、爪が刺さる。そして、血走った目で暁の胸のうちを射抜くように、夏美は深く深く顔を近づけてきた。",

> 			NULL, "...Her nails dug into his throat with a sound. Akira's eyes went bloodshot as Natsumi moved her face closer and closer as if trying to pierce his chest.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Her nails audibly dug into his throat. Akira's eyes went bloodshot as Natsumi moved her face closer and closer, as if trying to pierce his chest.", Line_Normal);
```

//r暁の背中があわ立ち、全身の血管が凍りついたように寒気がかけ巡る。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "暁の背中があわ立ち、全身の血管が凍りついたように寒気がかけ巡る。",

> 			NULL, "Goosebumps erupted from Akira's back. His blood vessels seemeed to freeze over as chills ran all over through his body.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Goosebumps erupted from Akira's back. His blood vessels seemed to freeze over as a chill ran all through his body.", Line_Normal);
```

//r感情も感覚も全て失い、もはや空虚の塊と成り果ててしまったが、それでも、…目の前の惨劇を受け入れられない衝撃が、彼の瞬きを阻んでいた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "感情も感覚も全て失い、もはや空虚の塊と成り果ててしまったが、それでも、…目の前の惨劇を受け入れられない衝撃が、彼の瞬きを阻んでいた。",

> 			NULL, "He was an empty shell completely devoid of any remaining sensations and emotions. But even so... the shock from his total denial of the tragedy that unfolded before him left him unable to blink.", Line_Normal);

Reviewer: ```suggestion
			NULL, "He was an empty shell completely devoid of any remaining sensations and emotions. But even so... the shock from his total denial of the tragedy that unfolded right before him left him unable to blink.", Line_Normal);
```

//r夏美の腕を掴んでいたその手は、…影を伴って浮動し、……やがて、離れる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "夏美の腕を掴んでいたその手は、…影を伴って浮動し、……やがて、離れる。",
			NULL, "His hands, grabbing at Natsumi's arms... floated along with his shadow...... and eventually separated from her.", Line_Normal);
	ClearMessage();

//rそれを見届けると、夏美は一瞬だけ、…穏やかな微笑を浮かべた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それを見届けると、夏美は一瞬だけ、…穏やかな微笑を浮かべた。",

> 			NULL, "Then, for only a moment... he saw Natsumi give a gentle smile.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Then, for only a moment... he saw Natsumi don a gentle smile.", Line_Normal);
```

//rそう。——夏美が豹変した瞬間に、暁の心は彼女を見捨てて、…逃げ出していた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう。——夏美が豹変した瞬間に、暁の心は彼女を見捨てて、…逃げ出していた。",

> 			NULL, "That was right. ...Akira's heart abandoned Natsumi the moment she suddenly changed... and fled from her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "That was right. ...Akira abandoned Natsumi in his heart the moment she suddenly changed... and he fled from her.", Line_Normal);
```

//rこの結末を迎えることを、夏美はどこかで予感していたのだろうか…？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "この結末を迎えることを、夏美はどこかで予感していたのだろうか…？",

> 			NULL, "He wondered if Natsumi had foreseen things would end this way...?", Line_Normal);

Reviewer: ```suggestion
			NULL, "He wondered if Natsumi expected things to end this way...?", Line_Normal);
```

//r自分に言い聞かせるように、そう呟き続ける。その言葉にもう、…一番答えて欲しい人は二度と言葉を発しないということを、十分すぎるほどに理解していたけれど…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "自分に言い聞かせるように、そう呟き続ける。その言葉にもう、…一番答えて欲しい人は二度と言葉を発しないということを、十分すぎるほどに理解していたけれど…。",

> 			NULL, "He continued muttering those words in an attempt to convince himself. Even though he was painfully aware... the one he wanted to answer him the most would never speak again.", Line_Normal);

Reviewer: ```suggestion
			NULL, "He continued muttering those words in an attempt to convince himself. Even though he was painfully aware... the one he most wanted to answer him would never speak again.", Line_Normal);
```

//r怒りを抑えきれず、車のボンネットを殴りつける。……こぶしの先から血がにじんで赤く染まったが、その痛みすら、心のそれに比べても大した傷ではなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "怒りを抑えきれず、車のボンネットを殴りつける。……こぶしの先から血がにじんで赤く染まったが、その痛みすら、心のそれに比べても大した傷ではなかった。",

> 			NULL, "Unable to control my anger, I punched the hood of the car. ......I could my fist gushing with blood, but the pain of that wound was nothing compared to the pain in my heart.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Unable to control my anger, I punched the hood of the car. ......I could see my fist gushing with blood, but the pain of that wound was nothing compared to the pain in my heart.", Line_Normal);
```

//rまた、守れなかった…！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "また、守れなかった…！",
			NULL, "Again! Again I failed to protect them...!", Line_Normal);
	ClearMessage();

//rこれで最後にするって、…雛見沢の悲劇は最後にするって約束したのに！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "これで最後にするって、…雛見沢の悲劇は最後にするって約束したのに！！",
			NULL, "I swore this would be the end... I swore this would be the end of the tragedy in Hinamizawa!!", Line_Normal);
	ClearMessage();

//rなにが、約束を破るのは嫌い、だ！　笑わせるなッ！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "なにが、約束を破るのは嫌い、だ！　笑わせるなッ！！",

> 			NULL, "What was all that crap you talked about hating breaking promises! What nerve you have, Tomoe Minai!!", Line_Normal);

Reviewer: ```suggestion
			NULL, "What was all that crap you said about hating to break promises! What nerve you have, Tomoe Minai!!", Line_Normal);
```
